assets/pages/contributions-landing/components/contributionsBundle.jsx

```diff
@@ -77,6 +80,13 @@ const getContribAttrs = ({ contribType, contribAmount, intCmp }): ContribAttrs =
 
   const ctaLink = `${ctaLinks[contType]}?${params.toString()}`;
 
+  if (!addingMonthlyTest) {
```

Comment:
I would rename this variable to something more meaningful like `inOneoffAndMonthlyVariant`.